helm-chart/values.yaml

```diff
@@ -31,6 +31,7 @@ args: []
 tracing:
   # tracing is only enabled if below field is configured true
   enabled: false
+  port: 9202
```

Comment:
I think Ramon mentioned that we plan on ingesting metrics via OTLP. In that world, the OTLP interface services both tracing and metrics, so we would probably have to move this configuration at that point anyway.